docs/develop/gettingstarted.rst

```diff
@@ -314,15 +314,15 @@ extmod
 
 docs
 
-  Has the standard documentation found at https://docs.micropython.org/.
+  Has the standard documentation found at https://docs.micropython.org/en/latest/.
```

Comment:
I think the original may have been more correct?

Reply:
yeah I think you're right here, it would be a redirect onto the relevant version or latest. Good call.